resources/sass/materialize.scss

```diff
@@ -68,6 +68,16 @@ span.badge.tag {
     float: none;
 }
 
+// for the list of workshop functionaries on the home page
+a#workshop-administrator-collapsible {
+    color: inherit; text-decoration: inherit;
+}
+.dropdown-triangle {
+    display: inline-block; font-size: 0.6em; transform: translateY(-0.4em);
+}
```

Comment:
You can also add the icon content with CSS, so we don't have to specify it in HTML. 
You can also check if another class also exists on an element, it displays the other triangle.

You should be able to combine these to get a HTML structure like this. When you click on the div, the toggle functions toggles the "hide" class on the content while updating the icon. 
```
<div class="dropdown"> //display the corresponding icon, this has an event listener with the toggle function
<div class="dropdown-title">Title</div>
<div class="dropdown-content">Some content that may be hidden</div>
</div>
```

Reply:
And I would add the icon after the title, it can mess up with the margins on the left side.

Reply:
And if you do this, don't forget to add the .on('click', ...) listener in the main app layout page, next to the other initialisers.